content/en/blog/call-for-blogs.md

```diff
@@ -0,0 +1,42 @@
+# Call for blog contributions
```

Comment:
This is not how to start a blog article. Check the existing articles to see how it's done.